sync/patches/values/values.yaml.tmpl

```diff
@@ -1348,7 +1348,9 @@ hubble:
     # ref: https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/
     tolerations: []
     # -- Additional hubble-relay environment variables.
-    extraEnv: []
+    extraEnv:
+      - name: GOPS_CONFIG_DIR
+        value: /tmp
```

Comment:
How did you figure that one?

Reply:
gerald did. I'm bringing what he did in https://github.com/giantswarm/cluster-aws/pull/734